packages/wallets/src/components/WalletMarketCurrencyIcon/WalletMarketCurrencyIcon.tsx

```diff
@@ -19,11 +20,16 @@ const WalletMarketCurrencyIcon: FC<TWalletMarketCurrencyIconProps> = ({
     isDemo,
     marketType,
     platform,
+    product,
     size = 'sm',
 }) => {
     let MarketTypeIcon;
-    if (marketType === MARKET_TYPE.ALL && platform && platform in CFDPlatformIcons) {
-        MarketTypeIcon = platform;
+    if (marketType === MARKET_TYPE.ALL && platform) {
+        if (platform in CFDPlatformIcons) {
+            MarketTypeIcon = platform;
+        } else {
+            MarketTypeIcon = `${marketType}_${product}`;
```

Comment:
why not include this in `CFDPlatformIcons`?

Reply:
added..